### PR TITLE
Run the entire analysis together

### PR DESCRIPTION
**Pull request by runnersaw**
Runs the entire analysis by:
- Runs /analysis and keeps polling until `safety_analysis` is done.
- Then, runs `api.results` and keeps polling until `highlight_video` is done.
- Then downloads the results, unzips them, and opens the file location

### CODE REVIEW

**Comment by youralien** [comment]
This is what the GUI looks like when I test on the SCOPE mac.  I can see what I can change in the .ui file to fix this.

screen shot 2017-02-13 at 2 20 38 pm

**Comment by youralien** [comment]
I made a PR that fixes the button size here: https://github.com/santosfamilyfoundation/SantosGUI/pull/46

File: application/app.py [hunk 1]
```diff
@@ -140,14 +139,35 @@ def runAnalysis(self):
...
+        with zipfile.ZipFile(os.path.join(results_dir, "results.zip"), "r") as zip_file:
+            zip_file.extractall(results_dir)
+
+        # Open the file location
+        if sys.platform == 'darwin':
+            subprocess.Popen(['open', '--', results_dir])
+        elif sys.platform == 'linux2':
+            subprocess.Popen(['xdg-open', '--', results_dir])
+        elif sys.platform == 'win32':
+            subprocess.Popen(['explorer', results_dir])
```
Comment (dcelik) [inline application/app.py:168]: Cross platform compatibility :) 👍

**Review by dcelik — APPROVED** [review]
Looks good! Haven't tested it on my mac yet but the PR looks good.

**Comment by runnersaw** [comment]
Going to merge. Just going to `dev`, not master, and I feel that it's fine to merge.

**Pull request merged**